controllers/grid/users/reviewer/ReviewerGridRow.php

```diff
@@ -215,6 +216,66 @@ public function initialize($request, $template = null)
                     )
                 );
             }
+            if ($reviewAssignment->getDateConfirmed() == null) {
+                $context = $request->getContext();
+                $vueModalArgs = [
+                    'description' => $submission->getCurrentPublication()->getLocalizedTitle(null, 'html'),
+                    'logResponseForm' => [
+                        'id' => 'logResponseForm',
```

Comment:
We do have classes to create form structure. Example would be https://github.com/pkp/pkp-lib/blob/main/classes/components/forms/dashboard/SubmissionFilters.php  or any other form in that forms folder.